aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -238,8 +249,8 @@ bool check_mem_efficient_hardware_support(sdp_params const& params, bool debug)
     }
     return false;
   }
-#endif
   return true;
+#endif
```

Comment:
Can you move #endif line before the return statement.
Otherwise it gives an error when compiled with debug mode - Control reaching end of non-void function